app/assets/stylesheets/fae/base.scss

```diff
@@ -67,6 +67,7 @@
   'modules/forms/asset-actions',
   'modules/forms/select',
   'modules/forms/date',
+  'modules/forms/colorpicker',
```

Comment:
My two cents but this looks like a stylesheet provided by the library. The styles in this folder are meant to override the default stylesheet provided by a library (the best example is how `simplemde.min.css` and `_simple-mde.scss` work together; the worst example is `_select.scss`). That way, when an upgrade is necessary, the CSS can be quickly updated and the modifications to integrate with Fae's appearance remain untouched.

If this isn't a library-provided CSS file (it's hard to tell from tinyColorPicker's repo/demo), please 
- avoid [over-qualifying elements as selectors](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#qualifyingelement)
- use [Fae's variables for grays](https://github.com/wearefine/fae/blob/master/app/assets/stylesheets/fae/globals/imports/_variables.scss#L47-L55)
- if you have to use an `!important`, make a note why